app/discover/procurement/page.tsx

```diff
@@ -0,0 +1,361 @@
+
+<Text>
+In other cases a certain functionality is in heavy use and other organisations might have similar wishes to develop it further, but that requires communication between organisations.
+</Text>
+
+<Text>
+On the other hand, the functionality might not be used by anyone and hasn&apos;t been maintained, so the project steering committee of Oskari is about to drop it out of the development. 
+</Text>
+
+<Text>
+The easiest way to find out where Oskari is heading next, see Oskari&apos;s roadmap.
+</Text>
+
+<h3>Creating new functionalities</h3>
+
+<Text>
+Before creating new functionalities, see the list of existing functionalities in the documentation <Link href='/documentation/docs/latest/5-Frontend-main-functionalities#Frontend-main-functionalities'>sections 5</Link> and <Link href='/documentation/docs/latest/6-Other-functionalities#Other-functionalities'>section 6</Link> and <a href='https://github.com/oskariorg/oskari-frontend-contrib'>the functionalities and configurations done by other Oskari users (the unofficial Oskari bundles created by the Oskari community).</a> One should also think about joint development: could this be something that some other organisation using Oskari could also wish to have?
+</Text>
+
+<Text>
+If you want to create new functionalities for Oskari, it might be a good idea to gather together all the ideas that you have and hand them over to the development team. Thus they can see which are top of your priorities now and what kind of things to expect next - and also suggest to you which of your ideas could be fulfilled at the same time quite easily. By coming up with multiple ideas at once they can also take the upcoming ideas into consideration when developing the first new functionalities.
+</Text>
+
+<Text>
+With new functionalities, one should remember that they must be maintained. When you create new functionalities to your Oskari service, remember to inform NLS both beforehand and after the development has taken place. By contacting the NLS FI&apos;s Oskari technical team before you start the development, your developers can get best practices for the development process and further ideas on how to create the functionalities that you want. 
+</Text>
+
+<Text>
+When the new functionalities are in use, one should publish the code in a GitHub repository and inform the Oskari technical team - this helps them to take into account the dependencies of the new functionality when they are making new versions of Oskari. There might be dependencies, libraries or functionalities in the Oskari core that are affect the new functionality, and if you don’t inform the Oskari technical team about what you are doing or what you have done, it is possible that something gets broken when it&#39;s the time to update the Oskari instance next time.
+</Text>
+
+<Text>
+The changes into Oskari should never be done into the core of Oskari but as a separate whole. This makes the updating of the Oskari core much easier in the future.
```

Comment:
Kind of weird way of saying: if you fork it and branch off, it will be terrible to maintain. So if you want something maintained, the best bet is to have it on the Oskari repositories ("core of Oskari"). The next best solution is to have it as application specific code on the application repository.

What you don't want is having your own branched off fork of Oskari that might be difficult to update as the official repositories have changes that conflict with ones on the branched off fork.